templates/js-scrape-single-page/main.js

```diff
@@ -10,6 +10,7 @@ await Actor.init();
 
 // Structure of input is defined in input_schema.json
 const input = await Actor.getInput();
+if (!input) throw new Error("Input is missing!");
```

Comment:
Maybe we can use an input schema default instead of this .... redundant line? 😅

Reply:
It's because of TS complains - to keep the js template exactly the same as the ts one.

Reply:
Yeah, thought so. What about typing the input instead of this check?

Reply:
All right, I'll remove the check.